test/test_helper.rb

```diff
@@ -1,10 +1,17 @@
 require 'stripe'
+require 'committee'
+require 'json_schema'
 require 'test/unit'
 require 'mocha/setup'
-require 'stringio'
 require 'shoulda'
+require 'sinatra'
+require 'stringio'
+require 'webmock/test_unit'
+
 require File.expand_path('../test_data', __FILE__)
 
+$execute_request_stub = true
```

Comment:
>Can you make this a module-level ivar under the Stripe module rather than a global? (I guess you'll need an attr_accessor as well.)

+1. Done in 2afc076.